src/shared/seccomp-util.c

```diff
@@ -224,6 +225,16 @@ int seccomp_init_for_arch(scmp_filter_ctx *ret, uint32_t arch, uint32_t default_
         if (r < 0)
                 goto finish;
 
+#ifdef SCMP_FLTATR_CTL_LOG
+        if (seccomp_api_get() >= 3) {
+                if (getenv_bool("SYSTEMD_LOG_SECCOMP")) {
+                        r = seccomp_attr_set(seccomp, SCMP_FLTATR_CTL_LOG, 1);
+                        if (r < 0)
+                                log_debug_errno(r, "Failed to enable seccomp event logging");
+                }
+        }
```

Comment:
we try to keep indentation levels minimal, hence merge the two if expressions into one?